src/net_processing.cpp

```diff
@@ -2490,6 +2499,27 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             pfrom->nPingNonceSent = 0;
         }
     }
+    
+    
+    else if (strCommand == NetMsgType::BFD)
```

Comment:
Right now the bitcoin core client doesn't really support BFD for anything, since it's either a pruned node (in which case it has validated already, and is getting full blocks for new blocks) or it's a full node (in which case it has all blocks), so there's really no use case for bitcoin core itself here just like for bloom filters.